### PR TITLE
add missing file

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/config/test_data.config [hunk 1]
```diff
@@ -125,6 +125,7 @@ params {
                 genome_gff3                                    = "${test_data_dir}/genomics/homo_sapiens/genome/genome.gff3"
                 genome_gtf                                     = "${test_data_dir}/genomics/homo_sapiens/genome/genome.gtf"
                 genome_interval_list                           = "${test_data_dir}/genomics/homo_sapiens/genome/genome.interval_list"
+                genome_multi_interval_bed                      = "${test_data_dir}/genomics/homo_sapiens/genome/multi_intervals.bed"
```
Comment (JoseEspinosa) [inline tests/config/test_data.config:128]:
```suggestion
               genome_multi_interval_bed                      = "${test_data_dir}/genomics/homo_sapiens/genome/genome.multi_intervals.bed"
```

**Review by JoseEspinosa — APPROVED** [review]
I suggested changing the name but up to you :+1:

**Comment by maxulysse** [comment]
> I suggested changing the name but up to you :+1: 

I don't understand your suggestions

**Comment by maxulysse** [comment]
@nf-core-bot prettier lint please 🥺

**Review by JoseEspinosa — APPROVED** [review]
I was referring to the name of the file but this can not be changed here and it's not so important 🤓

**Pull request merged**